test/unit_tests/braket/default_simulator/test_observables.py

```diff
@@ -55,16 +62,44 @@
 y_diag = np.array([[1, -1j], [1, 1j]]) / np.sqrt(2)
 
 
-@pytest.mark.parametrize("observable, targets, measured_qubits, eigenvalues, is_standard", testdata)
-def test_observable_properties(observable, targets, measured_qubits, eigenvalues, is_standard):
-    if observable.diagonalizing_matrix is not None:
-        check_unitary(observable.diagonalizing_matrix)
-    assert observable.targets == targets
+@pytest.mark.parametrize(
+    "obs_class, extra_args, expected_gates, eigenvalues, is_standard", testdata
+)
+def test_observable_properties_single_qubit(
+    obs_class, extra_args, expected_gates, eigenvalues, is_standard
+):
+    num_qubits = np.random.randint(1, 100)
```

Comment:
I'd think we probably don't want to make the tests nondeterministic. What's wrong with simply picking a large number here (and later in the all_qubits test too)?

Reply:
No problems with that.